FRP/Elerea/Simple/Compat.hs

```diff
@@ -0,0 +1,11 @@
+{-# LANGUAGE CPP #-}
+module FRP.Elerea.Simple.Compat
+    ( module FRP.Elerea.Simple
+    , module FRP.Elerea.Simple.Compat
+    ) where
+import FRP.Elerea.Simple
+
+#if !MIN_VERSION_elerea(2, 9, 0)
+till :: Signal Bool -> SignalGen (Signal Bool)
+till = unless
```

Comment:
Oops. Good catch. I should fix the Travis CI to run tests on multiple versions of GHC.